linters/ecs.php

```diff
@@ -43,38 +43,41 @@
 use Symplify\EasyCodingStandard\ValueObject\Set\SetList;
 
 return static function (ECSConfig $containerConfigurator): void {
-    $containerConfigurator->import(SetList::PSR_12);
-    $containerConfigurator->import(SetList::DOCTRINE_ANNOTATIONS);
+    $containerConfigurator->sets([
+        SetList::PSR_12,
+        SetList::DOCTRINE_ANNOTATIONS,
+    ]);
 
-    $services = $containerConfigurator->services();
-    $services->set(AlignMultilineCommentFixer::class);
-    $services->set(ArrayIndentationFixer::class);
-    $services->set(CombineConsecutiveIssetsFixer::class);
-    $services->set(CombineConsecutiveUnsetsFixer::class);
-    $services->set(ConcatSpaceFixer::class)->call('configure', [[
-        'spacing' => 'one',
-    ]]);
-    $services->set(DeclareEqualNormalizeFixer::class)->call('configure', [[
+    $containerConfigurator->rule(AlignMultilineCommentFixer::class);
+    $containerConfigurator->rule(ArrayIndentationFixer::class);
+    $containerConfigurator->rule(CombineConsecutiveIssetsFixer::class);
+    $containerConfigurator->rule(CombineConsecutiveUnsetsFixer::class);
+    $containerConfigurator->ruleWithConfiguration(ConcatSpaceFixer::class,
+        [
+            'spacing' => 'one',
+        ],
+    );
+    $containerConfigurator->ruleWithConfiguration(DeclareEqualNormalizeFixer::class, [
```

Comment:
Something is wrong here. This rule should be removed `DeclareEqualNormalizeFixer` to achieve
`declare(strict_types=1);`

SetList::PSR_12 already has this rule without spaces:
```
$ecsConfig->ruleWithConfiguration(DeclareEqualNormalizeFixer::class, ['space' => 'none']);
```